chapters/usage-profile.md

```diff
@@ -0,0 +1,326 @@
+
+With this "Usage Scope" field, the intended usage assumed by the package supplier is appended to the generated package in a manner that supplements those conditions contained in the source code.
+
+
+Table 31.1 — Metadata for the usage scope field
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..* |
+| Format | ["DocumentRef-"[idstring]":"]SPDXID \<single line of text\> \| `NONE` \| `NOASSERTION`<br>where "DocumentRef-"`[idstring]`":" is an optional reference to an external SPDX document as described in [6.6](document-creation-information.md#6.6)<br>where `SPDXID` is a string containing letters, numbers, `.` and/or `-`. to designate SPDX document/package/file as described in [6.3](document-creation-information.md#6.3), [7.2](package-information.md#7.2), [8.2](file-information.md#8.2).<br>where `<single line of text>` is the intended usage of the package assumed by the package supplier.<br>where `NONE` can be used to explicitly indicate there are NO other "usage scope".<br>where `NOASSERTION` can be used to explicitly indicate it is not clear if there are "usage scope" that may apply or not. |
+
+<br>
+
+### 31.1.2 Intent
+
+To describe detail intended usage of the package assumed by the package supplier in a manner that supplements those conditions contained in the software.
+
+### 31.1.3 Examples
+
+EXAMPLE 1 Tag: `UsageScope:`
+
+```text
+UsageScope: SPDXRef-a_specific_binary_verNNN: shipped for consumer product/ medical system / automotive system
+```
+
+EXAMPLE 2 RDF: Property `spdx:UsageScope` in any `spdx:SpdxDocument`, `spdx:Package` or `spdx:File`
+
+```text
+    <UsageScope>
+      <UsageScopeElement>
+        <File rdf:about="http://spdx.org/spdxdocs/spdx-tools-v1.2-3F2504E0-4F89-41D3-9A0C-0305E82..."
+      </UsageScopeElement>
+      <Usage>SPDXRef-a_specific_binary_verNNN: shipped for consumer product/ medical system / automotive system
+      </Usage>
+    </UsageScope>
+  ...
+
+```
+
+## 31.2 Usage Scope Comment field <a name="31.2"></a>
+
+### 31.2.1 Description
+
+This field provides a place for the SPDX document creator to record any general comments about the "usage scope".
+
+Table 31.2 — Metadata for the usage scope comment field
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..1 |
+| Format | Free form text that may span multiple lines, refers only to the immediately preceding usagescope.  |
+
+
+### 31.2.2 Intent
+
+To provide additional context. For example, describing prerequisite assumptions of usage scope by the package supplier.
```

Comment:
What's the difference between the text that is supposed to go into the `UsageScope` field vs. the `UsageScopeComment` field? Given that they are both just unstructured free text fields, would it be easier to just have text in the `UsageScopeComment` field and to have the `UsageScope` contain just the relevant SPDX ID for this section?